### ISSUE TITLE
Expansion of server-config default cracker command-line parameters per task

### ISSUE THREAD

**Issue by thatux** [hashtopolis/server#785]
Currently it's not possible to edit `default cracker command-line parameters` before starting a new task. 
If that would be possible one could add -O or -S to Hashcat by default, but remove that option when need be.

One way to achieve this is to do the implement the following:

Instead of only filling in #HL# when creating a new task: /hashtopolis/tasks.php?new=true
![10](https://user-images.githubusercontent.com/2683920/172592525-c95124b9-2532-4072-82cb-9688e2c872af.png)

It'd be great if default cracker command-line parameters would also be added:
![20](https://user-images.githubusercontent.com/2683920/172592563-ec2541ce-58f8-4140-8152-0c8eb7e416d4.png)

Currently there is no server-config option for `default cracker command-line parameters`, as this is set per agent: /hashtopolis/agents.php?id=1
![30](https://user-images.githubusercontent.com/2683920/172592609-df22ab57-b682-4c08-8705-9bda8af39d80.png)

The current server-config option is e.g. only #HL# /hashtopolis/config.php 
![40](https://user-images.githubusercontent.com/2683920/172592711-de1edfb7-328a-4e6c-9b34-1b4b00cf8f0b.png)

Therefore such an server-config option would need to be added:
![50](https://user-images.githubusercontent.com/2683920/172592686-9e2bb01f-9128-4bdd-aa14-f882928ca0d9.png)

**Comment by zyronix** [comment]
Hmm, make sense and I think it is quite easy to implement and useful.

In tasks.php the following code is called:
```
  /**
   * @return Task
   */
  public static function getDefault() {
    return new Task(
      null,
      "",
      "",
      SConfig::getInstance()->getVal(DConfig::CHUNK_DURATION),
      SConfig::getInstance()->getVal(DConfig::STATUS_TIMER),
      0,
      0,
      0,
      "",
      0,
      0,
      SConfig::getInstance()->getVal(DConfig::DEFAULT_BENCH),
      0,
      0,
      0,
      0,
      0,
      '',
      0,
      0,
      0,
      0,
      ''
    );
  }
```
Just add a getVal from the config here

Also the lines 364 in tasks.php should not cause an issue:
```
  if (strpos($copy->getAttackCmd(), SConfig::getInstance()->getVal(DConfig::HASHLIST_ALIAS)) === false) {
    $copy->setAttackCmd(SConfig::getInstance()->getVal(DConfig::HASHLIST_ALIAS) . " " . $copy->getAttackCmd());
  }
```

I am pretty sure I'll have some time in the upcoming weeks for this, but if anyone else wants to implement this, be my guest (just let me know that you are going to ;)).
